tsconfig.json

```diff
@@ -18,5 +18,9 @@
     "tasks/enableL1TokenAcrossEcosystem.ts",
     "utils/utils.ts"
   ],
+  "exclude": [
```

Comment:
the new ts associated with sim stuff is not yet totally type safe! we are simply ignoring this for now and will improve it soon